src/helpers/FunctionHelpers.js

```diff
@@ -0,0 +1,58 @@
+const diceNumberToSymbol = {
+    1: 'coin',
+    2: 'diamond',
+    3: 'sword',
+    4: 'parrot',
+    5: 'monkey',
+    6: 'skull',
+};
+
+const getRandomDiceNumber = () => Math.floor(Math.random() * 6) + 1;
+
+const getRollDiceResults = (diceNumber) => {
+    const resultArray = [];
+    for (let i = 0; i < diceNumber; i++) {
+        const diceNumber = getRandomDiceNumber();
+        resultArray.push(diceNumberToSymbol[diceNumber]);
+    }
+    return resultArray;
+}
+
+const countSymbolsOccurences = arr => {
```

Comment:
Should we speak english or french on github?

As discussed let's make this function easier to understand and more specific to our use case.
I suggest something like below fell free to change names or behaviour, it's just an example.

```suggestion
const countSymbolsOccurences = diceResultArray => {
  const symbolCountMap = {}
  diceResultArray.forEach(diceResult => {
    if (symbolCountMap.hasOwnProperty(diceResult)) {
      symbolCountMap[diceResult]++
    }
    else {
      symbolCountMap[diceResult] = 1
    }
  })
  return symbolCountMap
}
```